ui-tests/playwright.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
   },
   expect: {
     toMatchSnapshot: {
-      maxDiffPixelRatio: 0.0001
+      maxDiffPixelRatio: 0.01
```

Comment:
```suggestion
      maxDiffPixelRatio: 0.001
```
1% might be overkill

Reply:
It was not, I'm reverting that change.